R/pkg/R/mllib.R

```diff
@@ -266,9 +266,9 @@ setMethod("summary", signature(object = "NaiveBayesModel"),
             return(list(apriori = apriori, tables = tables))
           })
 
-#' Fit a k-means model
+#' K-Means Model
```

Comment:
sorry I actually meant the description could be more like plain sentences, rather than bullet points.